src/unread/unreadModel.js

```diff
@@ -230,13 +230,6 @@ function streamsReducer(
       const { stream_id } = action;
       if (stream_id == null) {
         // Not stream messages, or else a pure content edit (no stream/topic change.)
```

Comment:
That's right. I wouldn't want to remove that part (because then the comment wouldn't necessarily be true, which risks misleading us when making some future change); but adding a TODO(server-5.0) seems helpful for making it clear how it relates to that API change.